tools/tcpcong_example.txt

```diff
@@ -0,0 +1,463 @@
+       256 -> 511        : 14852    |****************************************|
+       512 -> 1023       : 13753    |*************************************   |
+      1024 -> 2047       : 3224     |********                                |
+      2048 -> 4095       : 598      |*                                       |
+      4096 -> 8191       : 41       |                                        |
+      8192 -> 16383      : 0        |                                        |
+     16384 -> 32767      : 1        |                                        |
+     32768 -> 65535      : 0        |                                        |
+     65536 -> 131071     : 0        |                                        |
+    131072 -> 262143     : 1        |                                        |
+
+tcp_congest_state = loss
+     usecs               : count     distribution
+         0 -> 1          : 0        |                                        |
+         2 -> 3          : 0        |                                        |
+         4 -> 7          : 0        |                                        |
+         8 -> 15         : 0        |                                        |
+        16 -> 31         : 0        |                                        |
+        32 -> 63         : 0        |                                        |
+        64 -> 127        : 1        |******                                  |
+       128 -> 255        : 0        |                                        |
+       256 -> 511        : 2        |*************                           |
+       512 -> 1023       : 6        |****************************************|
+      1024 -> 2047       : 0        |                                        |
+      2048 -> 4095       : 0        |                                        |
+      4096 -> 8191       : 0        |                                        |
+      8192 -> 16383      : 0        |                                        |
+     16384 -> 32767      : 0        |                                        |
+     32768 -> 65535      : 1        |******                                  |
+
+
+USAGE:
+./tcpcong -h
+usage: tcpcong [-h] [-L LOCALPORT] [-R REMOTEPORT] [-T] [-d] [-u]
+                  [interval] [outputs]
+
+Summarize tcp socket congestion control status duration
+
+positional arguments:
+  interval              output interval, in seconds
+  outputs               number of outputs
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -L LOCALPORT, --localport LOCALPORT
+                        trace local ports only
+  -R REMOTEPORT, --remoteport REMOTEPORT
+                        trace the dest ports only
+  -T, --timestamp       include timestamp on output
+  -d, --dist            show distributions as histograms
+  -u, --microseconds    output in microseconds
+
+examples:
+    ./tcpcong                 #show tcp congestion status duration
```

Comment:
Let us put a space between '\#' and 'show'. The same for other examples below.

Reply:
okay, will update it